### ISSUE TITLE
Exception when doctrine/data-fixtures is not installed

### ISSUE THREAD

**Issue by teohhanhui** [doctrine/DoctrineMongoDBBundle#557]
```
                                                                                                                        
 [WARNING] Some commands could not be registered:                                                                       
                                                                                                                        

In ContainerAwareLoader.php line 26:
                                                           
  [Symfony\Component\Debug\Exception\FatalThrowableError]  
  Class 'Doctrine\Common\DataFixtures\Loader' not found    
                                                           
```

This is caused by the service `doctrine_mongodb.odm.symfony.fixtures.loader` (https://github.com/doctrine/DoctrineMongoDBBundle/blob/4.0.0-RC1/Loader/SymfonyFixturesLoader.php) which should not be registered if the `doctrine/data-fixtures` package is not installed.

**Comment by smoench** [comment]
@alcaeus just a reminder #559 must be ported/merged to master before releasing the next 4.0 version. When you got some time a 4.0-RC2 with those changes might be good for having another test run :-)

**Comment by alcaeus** [comment]
I know - merges to Master are periodical but always before a release 😉
